src/conditionals/programs_with_decision_points/medium/q1/MainTest.java

```diff
@@ -0,0 +1,46 @@
+package conditionals.programs_with_decision_points.medium.q1;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+    // Parsons
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "a";
+
+        return new Clause[]{
+                new StringLiteral("Enter a character: "),
+                new NewLine(),
+                new PlaceHolder()
+        };
+    }
+
+    public void runMain() {
+        LetterBox.main(new String[0]);
+    }
+
+    //check lower-case a and z for if statement, then uppercase, then random symbols
+    static Stream<Arguments> inputProvider() {
+        return Stream.of(Arguments.of("a", "a is a lowercase letter"), Arguments.of("z", "z is a lowercase letter"), Arguments.of("A", "A is an uppercase letter"), Arguments.of("Z", "Z is an uppercase letter"), Arguments.of("&", "& is neither uppercase not lowercase"), Arguments.of("/", "/ is neither uppercase not lowercase"));
```

Comment:
in this case you can just have `"lowercase"`/`"uppercase"` as the second argument and create the message during the test

Reply:
I can't actually do that right now bc when it is neither it doesn't follow with the word letter. Though I've also just noticed a spelling error in the question code. Do you want me to change the wording of the question code when I fix the spelling error so I can implement this?

Reply:
Just to document the solution from the meeting here, too, I think when the message differs this much it's a good time to use multiple input providers and test methods